engine_generative_react.ipynb

```diff
@@ -718,23 +820,23 @@
    "id": "f33b23a7",
    "metadata": {},
    "source": [
-    "We can see here that generative predictions with high conviction have higher accuracy than generative predictions with low conviction, while the discriminative predictions have the highest accuracy overall."
+    "We can see here that generative predictions with high conviction typically have higher accuracy than generative predictions with low conviction. Ideally, the discriminative predictions will have the highest accuracy, but it's not impossible that the generative predictions give more accurate predictions on untrained data."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "1722e91f",
    "metadata": {},
    "source": [
-    "# Section 2: Synthetic Data Generation\n",
+    "# Step 3: Synthetic Data Generation with Generative React\n",
     "\n",
-    "Generative analysis is also useful to create synthetic data, as it can generate entirely new points that follow the distibution of the original data.\n",
+    "Generative analysis is also useful to create synthetic data, as it can generate entirely new points that follow the distribution of the original data.\n",
     "\n",
-    "Synthetic data creation is similar to the generative analysis performed in Section 1, but uses a *targetless* optimization scheme. Additionally, you will set two additional parameters:\n",
+    "Synthetic data creation is similar to the generative analysis performed in Section 1, but we recommend using a *targetless* analysis. Additionally, there are two other parameters you should consider based on your use-case:\n",
     "\n",
-    "    -`generate_new_cases`: whether a completely new case is or is not always generated\n",
+    "- `generate_new_cases`: A string that specifies if generated feature values should represent completely new cases. \"no\" indicates that privacy of the training data is unimportant and generated cases can be similar to the training data. \"attempt\" and \"always\" will perform a uniqueness check after a set of case values are generated.  When using \"always\", only cases that pass the uniqueness check will be returned. When using \"attempt\", cases that fail the check will be retried, but the constraint will be lifted if it fails multiple times consecutively.\n",
```

Comment:
Did we want to be as specific here... I remember this recipe was supposed to be vague about synthesis, so maybe we shouldn't mention the specific synth parameters? Just a thought...

Reply:
Oh, I wasn't aware of the strategy here. Will lose the specificity.